docs/readmes/README.featureflags.md

```diff
@@ -22,23 +22,8 @@ node/VMSS.
 Aad-pod-identity has a new flag clientQps which can be used to control the total number of client operations performed per second
 to the API server by MIC.
 
-## Block Instance Metadata flag
```

Comment:
Looks like there is a need to rebase the code. The update to README from older patch got removed in this.

Reply:
Fixed